### DISCUSSION TITLE
Welcome to prenotami Discussions!

### DISCUSSION THREAD

**Discussion by handreassa** [handreassa/prenotami#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alantody** [comment]
Hello my name is Alan and i'm working in some changes on project
I made some modifications to the delays, and I obtained a good result, but I'm not sure if it's a good practice because I'm a junior developer and I started with Python recently.
I'm open to suggestions, and I hope that I have contributed somewhat to the project.

Follow the link to my fork> https://github.com/alantody/prenotami

@handreassa Thanks for the project!

**Comment by natanmondoni** [comment]
Olá! Tudo bem?
Você sabe me dizer se aquele script para o agendamento no Prenotami ainda está funcionando normalmente?
Fiquei na dúvida se alguém que usou teve algum problema depois — tipo perder acesso ao site ou ter algum impacto no processo de cidadania.
Agradeço se puder me confirmar! 🙏 @handreassa @alantody @borattoo
